lib/main.dart

```diff
@@ -11,13 +16,36 @@ class App extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    EditorViewController controller = EditorViewController(
-      options: const EditorOptions(
-          canCloseFiles: false, showAppBar: false, showTabBar: false),
+    Editor editor = Editor(
+      language: Syntax.HTML,
+      openedFile: FileIDE(
+        fileContent: 'hello',
+        fileExplorer: null,
+        fileName: '',
+        filePath: '',
+        parentDirectory: '',
+      ),
     );
 
+    EditorViewController controller = EditorViewController(
+        options: const EditorOptions(
+          canCloseFiles: false,
+          showAppBar: false,
+          showTabBar: false,
+        ),
+        editor: editor);
+
     return MaterialApp(
-      home: controller,
+      home: Row(
+        children: [
+          Expanded(child: controller),
+          TextButton(
+              onPressed: () {
+                editor.textStream.sink.add('this clearly work fine');
```

Comment:
This line is giving an analyzer error:

```
The getter 'textStream' isn't defined for the type 'Editor'.
Try importing the library that defines 'textStream', correcting the name to the name of an existing getter, or defining a getter or field named 'textStream'.dart - [undefined_getter](https://dart.dev/diagnostics/undefined_getter)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix